dev_guide/console.adoc

```diff
@@ -69,34 +69,3 @@ image:console_filter_step3.png["Console Filter Step 3"]
 
 image:console_filter_active.png["Console Filter Active"]
 ====
-
-
-== Creating from Template
```

Comment:
we probably still need some sort of link from the console doc to templates...  eg a high level summary of "here's all the stuff you can do in the console (link, link, link)"